docs/customization_guide/compose.md

```diff
@@ -74,32 +77,50 @@ script will extract components. The version of the `min` and `full` container
 is determined by the branch of Triton `compose.py` is on.
 For example, running
 ```
-python3 compose.py --backend tensorflow1 --repoagent checksum
+python3 compose.py --backend pytorch --repoagent checksum
 ```
 on branch [r24.03](https://github.com/triton-inference-server/server/tree/r24.03) pulls:
 - `min` container `nvcr.io/nvidia/tritonserver:24.03-py3-min`
 - `full` container `nvcr.io/nvidia/tritonserver:24.03-py3`
 
-Alternatively, users can specify the version of Triton container to pull from any branch by either:
+Alternatively, users can specify the version of Triton container to pull from
+any branch by either:
 1. Adding flag `--container-version <container version>` to branch
 ```
-python3 compose.py --backend tensorflow1 --repoagent checksum --container-version 24.03
+python3 compose.py --backend pytorch --repoagent checksum --container-version 24.03
 ```
 2. Specifying `--image min,<min container image name> --image full,<full container image name>`.
    The user is responsible for specifying compatible `min` and `full` containers.
 ```
-python3 compose.py --backend tensorflow1 --repoagent checksum --image min,nvcr.io/nvidia/tritonserver:24.03-py3-min --image full,nvcr.io/nvidia/tritonserver:24.03-py3
+python3 compose.py --backend pytorch --repoagent checksum --image min,nvcr.io/nvidia/tritonserver:24.03-py3-min --image full,nvcr.io/nvidia/tritonserver:24.03-py3
 ```
-Method 1 and 2 will result in the same composed container. Furthermore, `--image` flag overrides the `--container-version` flag when both are specified.
+Method 1 and 2 will result in the same composed container. Furthermore,
+`--image` flag overrides the `--container-version` flag when both are specified.
+
+Note:
+1. All contents in `/opt/tritonserver` repository of the `min` image will be
+ removed to ensure dependencies of the composed image are added properly.
+2. vLLM and TensorRT-LLM backends are currently not supported backends for
+`compose.py`. If you want to build additional backends on top of these backends,
+it would be better to [build it yourself](#build-it-yourself) by using
```

Comment:
@yinggeh @indrajit96 just double checking - do relative links like this work when creating the rendered HTML docs?

Reply:
It works as long as the hashtag section exists in this document.